citadel/indico_citadel/models/search_id_map.py

```diff
@@ -0,0 +1,229 @@
+        nullable=False,
+        default=now_utc
+    )
+
+    #: The type of the search entry object
+    entry_type = db.Column(
+        PyIntEnum(EntryType),
+        nullable=False
+    )
+
+    #: ID of the search entry event
+    event_id = db.Column(
+        db.Integer,
+        db.ForeignKey('events.events.id'),
+        index=True,
+        nullable=True
+    )
+
+    #: ID of the search entry contribution
+    contrib_id = db.Column(
+        db.Integer,
+        db.ForeignKey('events.contributions.id'),
+        index=True,
+        nullable=True
+    )
+
+    #: ID of the search entry subcontribution
+    subcontrib_id = db.Column(
+        db.Integer,
+        db.ForeignKey('events.subcontributions.id'),
+        index=True,
+        nullable=True
+    )
+
+    #: ID of the search entry attachment
+    attachment_id = db.Column(
+        db.Integer,
+        db.ForeignKey('attachments.attachments.id'),
+        index=True,
+        nullable=True
+    )
+
+    #: ID of the search entry note
+    note_id = db.Column(
+        db.Integer,
+        db.ForeignKey('events.notes.id'),
+        index=True,
+        nullable=True
+    )
+
+    #: ID of the search entry file
+    attachment_file_id = db.Column(
+        db.Integer,
+        db.ForeignKey('attachments.files.id'),
+        index=True,
+        nullable=True,
+    )
+
+    event = db.relationship(
+        'Event',
+        lazy=True,
+        backref=db.backref(
+            'citadel_es_entries',
+            cascade='all, delete-orphan',
+            lazy=True
+        )
+    )
+
+    contribution = db.relationship(
+        'Contribution',
+        lazy=False,
+        backref=db.backref(
+            'citadel_es_entries',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+
+    subcontribution = db.relationship(
+        'SubContribution',
+        lazy=False,
+        backref=db.backref(
+            'citadel_es_entries',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+
+    attachment = db.relationship(
+        'Attachment',
+        lazy=False,
+        backref=db.backref(
+            'citadel_es_entries',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+
+    note = db.relationship(
+        'EventNote',
+        lazy=False,
+        backref=db.backref(
+            'citadel_es_entries',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+
+    attachment_file = db.relationship(
+        'AttachmentFile',
+        lazy=False,
+        backref=db.backref(
+            'citadel_es_entries',
+            cascade='all, delete-orphan',
+            lazy='dynamic'
+        )
+    )
+
+    @classmethod
+    def get_search_id(cls, oid, obj_type):
+        """Get the search_id for a given object type and id.
+
+        :param oid: The id of the object
+        :param obj_type: the EntryType of the object
+        """
+
```

Comment:
there shouldn't be a blank line here 👾

Reply:
Indeed, this slipped through a couple commits.